### ISSUE TITLE
Disabel fix of DR X/-

### ISSUE THREAD

**Issue by dtndmd** [Vek17/WrathMods-TabletopTweaks#166]
Is it possible to turn off the Damage Reduction change in 1.4.0?
for example  Aivu, Azata's The Bound of Possibility,  Lich, Warden of Darkness,  Bruiser's Chainshirt

**Comment by bguns** [comment]
Yes. You can open the Fixes.json file with a text editor, find "DRRework", and in there change "Base": true to "Base": false.

The Fixes.json file is in <install directory>\Mods\TabletopTweaks\UserSettings
